schema/tables/login_password.yml

```diff
@@ -0,0 +1,12 @@
+name: login_password
```

Comment:
Let's clarify (or rename) that this table checks some settings for ALL users (in this PR we add a check to know if any of all the users in the device has a password hint set)

Reply:
'users_login_password' ?

Reply:
@sharon-fdm How about `user_login_settings`?

Reply:
If we agree the table name should be changed, we should change the table name now (and not rename later), because it's a pain to change later (users may start relying on it as soon as we release).